src/olympia/devhub/utils.py

```diff
@@ -298,3 +299,11 @@ def validate_upload(upload, channel, is_webextension):
             'listed': (channel == amo.RELEASE_CHANNEL_LISTED),
             'is_webextension': is_webextension}
         return tasks.validate_file_path.subtask([upload.path], kwargs)
+
+
+def add_dynamic_theme_tag(version):
+    if version.channel != amo.RELEASE_CHANNEL_LISTED:
+        return
+    files = version.files.all()
```

Comment:
Is there a reason not to re-use `version.all_files` if we're iterating through all files anyway?

Reply:
no reason, just forgot :)

Reply:
https://github.com/mozilla/addons-server/pull/8731/commits/5a75bc864ea11a284f0577b5e67031e6271d4468